### ISSUE TITLE
Faulty invitation code validation

### ISSUE THREAD

**Issue by aberonni** [growingabit/pwa-frontend#21]
**Steps to reproduce**
- Login
- Insert an invalid invitation code
- Proceed

**Expected result**
I should get an error message when I try to submit, telling me that the inserted code is invalid. The error message should be displayed before proceeding to step 2.

**Actual result**
The error message is shown once I am already on step 2, and I have no way of going back to step 1 and correcting the invitation code

**Notes**
- Ideally there should be some sort of client side validation before sending the data to backend. Eg. length of string. (Nice to have)
- As soon as I land on step 1 I can see the invalid code message. This should appear only once I start writing (Nice to have).

![image](https://user-images.githubusercontent.com/1260402/30980931-00822164-a483-11e7-9f80-6b72bf5ccd89.png)

**Comment by aberonni** [comment]
Fixed for me
